Sources/ApolloTestSupport/TimeInterval+Helpers.swift

```diff
@@ -0,0 +1,7 @@
+import struct Foundation.TimeInterval
+
+extension TimeInterval {
+  public static var oneHour: Self { 3600 }
+  public static var oneDay: Self { 86400 }
+  public static var oneYear: Self { Self.oneDay * 365 }
```

Comment:
Yup, you're definitely right that this is naïve. I only contributed it because of it being in tests - but having these as extensions for all tests leaves a loaded gun for a future developer writing unit tests that are needing to be more accurate.

I've updated references to use `Calendar`